.gitignore

```diff
@@ -348,3 +348,6 @@ MigrationBackup/
 
 # Ionide (cross platform F# VS Code tools) working folder
 .ionide/
+
+# Custom
+/AltinnReStorage/appsettings.json
```

Comment:
Why is appsettings.json added to .gitignore?

Reply:
It contains sensitive data such as subscription ids for all of our subscriptions.
We discussed making this file available on AltinnPedia or somewhere else.. Using mockdata and checking the file into the repo doesn't make much sense either